x-pack/plugins/cases/public/components/all_cases/index.test.tsx

```diff
@@ -851,7 +851,7 @@ describe('AllCases', () => {
     wrapper.find('button[data-test-subj="case-status-filter-in-progress"]').simulate('click');
     await waitFor(() => {
       expect(setQueryParams).toBeCalledWith({
-        sortField: 'createdAt',
+        sortField: 'updatedAt',
```

Comment:
the field in all_cases was `updatedAt`

Reply:
there is no column called updatedAt. there used to be but not anymore

Reply:
it is only ever closedAt or createdAt https://github.com/elastic/kibana/blob/f7423eb28231d13c380ec0111c1b2eed5e559a43/x-pack/plugins/cases/public/components/all_cases/columns.tsx#L164-L192

Reply:
Thanks! Was following this call here: https://github.com/elastic/kibana/blob/f7423eb28231d13c380ec0111c1b2eed5e559a43/x-pack/plugins/cases/public/components/all_cases/index.tsx#L363